platform-sdk/sdk/docs/installRaspbian.html

```diff
@@ -29,11 +29,11 @@
 </head>
 <body>
 
-<h1>Intalling Swirlds on Raspbian</h1>
+<h1>Installing Swirlds on Raspbian</h1>

```

Comment:
TIL that the platform runs on Raspbian!  (Or at least did at one time ...). There are Raspi5 with 64GB now and PCIe for NVMe drives ... wonder if a consensus node will run on one ... (maybe 64GB isn't sufficient though?)